cmd/jetstream-controller/main.go

```diff
@@ -37,7 +37,7 @@ import (
 var (
 	BuildTime = "build-time-not-set"
 	GitInfo   = "gitinfo-not-set"
-	Version   = "0.9.0"
+	Version   = "0.9.1"
```

Comment:
@samuelattwood changing this one to `not-set` would be ok since set by goreleaser?

Reply:
Correct